ngx-fudis/projects/ngx-fudis/src/lib/components/section/section.component.ts

```diff
@@ -67,12 +67,9 @@ export class SectionComponent
 
   /**
    * Maximum width of Grid. When viewport gets narrower, grid automatically adjusts to lower sizes.
-   * xxl = Default value. Viewports of 1600px and larger
-   * xl = Viewports smaller than 1600px
-   * lg = Viewports smaller than 1200px
-   * md = Viewports smaller than 992px
-   * sm = Viewports smaller than 768px
-   * xs = Viewports smaller than 576px
+   * xxl = Default value. Viewports of 1600px and larger xl = Viewports smaller than 1600px lg =
+   * Viewports smaller than 1200px md = Viewports smaller than 992px sm = Viewports smaller than
+   * 768px xs = Viewports smaller than 576px
```

Comment:
This is not readable. 
My suggestion is as follows: (JSDoc seems to allow linebreaks when using hyphens, but it also forces the first letter to be uppercase)

```
  /**
   * Maximum width of Grid. When viewport gets narrower, grid automatically adjusts to lower sizes.
   *
   * - Xxl = Default value. Viewports of 1600px and larger
   * - Xl = Viewports smaller than 1600px
   * - Lg = Viewports smaller than 1200px
   * - Md = Viewports smaller than 992px
   * - Sm = Viewports smaller than 768px
   * - Xs = Viewports smaller than 576px
   */
```